cmd/kubeadm/app/apis/kubeadm/v1alpha1/types.go

```diff
@@ -283,5 +283,9 @@ type AuditPolicyConfiguration struct {
 	LogDir string `json:"logDir"`
 	// LogMaxAge is the number of days logs will be stored for. 0 indicates forever.
 	LogMaxAge *int32 `json:"logMaxAge,omitempty"`
+	// WebhookConfigPath is the local path to webhook policy.
+	WebhookConfigPath string `json:"webhookConfigPath"`
```

Comment:
Do either of these have default you want to set in v1alpha1/defaults.go?

Reply:
Neither of them need a default.

The webhook config file must be provided.

Initial backoff has a default of 10 set in [apiserver/.../webhook.go](https://github.com/ii/kubernetes/blob/a491ee569c0bf413d57be380535dedac9ade2724/staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go#L42)

```go
	// DefaultInitialBackoff is the default amount of time to wait before
	// retrying sending audit events through a webhook.
	DefaultInitialBackoff = 10 * time.Second
```